interface/forms/misc_billing_options/new.php

```diff
@@ -13,27 +13,35 @@
  * @copyright Copyright (C) 2007 Bo Huynh
  * @copyright Copyright (C) 2016 Terry Hill <terry@lillysystems.com>
  * @copyright Copyright (C) 2017-2019 Brady Miller <brady.g.miller@gmail.com>
- * @copyright Copyright (C) 2017-2019 Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (C) 2017-2021 Stephen Waite <stephen.waite@cmsvt.com>
  * @copyright Copyright (C) 2018 Jerry Padgett <sjpadgett@gmail.com>
  * @copyright Copyright (c) 2019 Ranganath Pathak <pathak@scrs1.org>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-
 require_once(__DIR__ . "/../../globals.php");
 require_once("$srcdir/options.inc.php");
 require_once("$srcdir/api.inc");
 require_once("date_qualifier_options.php");
 require_once("$srcdir/user.inc");
+require_once("$srcdir/pid.inc");
+require_once("$srcdir/encounter.inc");
 
 use OpenEMR\Common\Csrf\CsrfUtils;
 use OpenEMR\Common\Session\SessionUtil;
 use OpenEMR\Core\Header;
 use OpenEMR\OeUI\OemrUI;
 
 if (isset($_REQUEST['isBilling'])) {
+    $is_from_billing_manager = true;
     $pid = $_SESSION['billpid'] = $_REQUEST['pid'];
```

Comment:
This should sort out the session issue (can not directly set a session variable; need to use this fancy function :) )
```
$pid = $_REQUEST['pid'];
SessionUtil::setSession('billpid', $pid);
```